docs/future-work/voting-improvements/can-we-change-the-cmp-to-use-displacements-instead-of-locations.md

```diff
@@ -6,4 +6,6 @@ Movement is core to how LMs process and model the world. Currently, an LM receiv
 
 However, a more general formulation might be to use displacements as the core spatial information in the CMP, such that a specific location (in body-centric coordinates or otherwise) is never communicated outside of an LM or sensor module.
 
-Such an approach might align well with adding information about flow (see [Detect Local and Global Flow](../sensor-module-improvements/detect-local-and-global-flow.md)), modeling moving objects (see [Deal With Moving Objects](../learning-module-improvements/deal-with-moving-objects.md)), and supporting abstract movements like the transition from grandchild to grandparent.
\ No newline at end of file
+Such an approach might align well with adding information about flow (see [Detect Local and Global Flow](../sensor-module-improvements/detect-local-and-global-flow.md)), modeling moving objects (see [Deal With Moving Objects](../learning-module-improvements/deal-with-moving-objects.md)), and supporting abstract movements like the transition from grandchild to grandparent. It would also result in a reformulation of "goal-states" to "goal-displacements".
+
+Note that whatever approach is taken, we would need to have information about the location or displacement of the sensor to ensure that communicated displacements are still in a shared coordinate system. This may relate to the division of "what" and "where" pathways in the brain, although this is not yet clear and requires further investigation.
```

Comment:
I don't see how displacements could be in a shared coordinate system. If we only communicate displacements, there is no way of knowing where the movements are relative to each other. I still think some information about location in a common reference frame needs to be present. maybe like you mention the difference between where and what pathways

Reply:
yeah for a single LM recognizing objects that works but I don't see how that works when multiple LMs start communicating or when we want to send goal states. How would you do voting or goal states without a shared reference frame of where sensors are relative to some common reference point?

Reply:
Hmm yeah that's a good point, now that I think about it more, it does feel like location is required for to some degree, I've updated the wording.